core/src/test/java/com/wechat/pay/java/core/notification/NotificationParserTest.java

```diff
@@ -55,14 +56,14 @@ public void testParse() throws IOException {
     verifyNonce = notification.getResource().getNonce();
 
     AeadCipher fakeAeadCipher =
-        new AeadCipher(verifyAlgorithm, TRANSFORMATION, KEY_LENGTH_BIT, KEY) {
+        new AbstractAeadCipher(verifyAlgorithm, TRANSFORMATION, KEY_LENGTH_BIT, KEY) {
```

Comment:
这里的 `fakeAeadCipher` 应该不需要借助 `AbstractAeadCipher`吧，直接用匿名类实现 `AeadCipher` 这个接口就可以？
```suggestion
        new AeadCipher() {
```